doc/man3/SSL_get_version.pod

```diff
@@ -38,6 +39,8 @@ SSL_is_tls() returns 1 if the connection is using SSL/TLS or 0 if not.
 
 SSL_is_quic() returns 1 if the connection is using QUIC or 0 if not.
 
+SSL_CTX_is_quic() returns 1 if the ctx creates QUIC connections or 0 if not.
```

Comment:
An SSL_CTX can create other types of QUIC objects besides connections (e.g. listeners).